gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/store/hdfs/SimpleHDFSConfigStoreFactory.java

```diff
@@ -38,6 +41,47 @@
   protected static final String SIMPLE_HDFS_SCHEME_PREFIX = "simple-";
   protected static final String HDFS_SCHEME_NAME = "hdfs";
 
+  public final static String DEFAULT_STORE_URI_KEY = "default_store_uri";
+
+  public SimpleHDFSConfigStoreFactory() {
+    this(ConfigFactory.empty());
+  }
+
+  private final Optional<URI> defaultStoreURI;
+  private final FileSystem defaultStoreFS;
+
+  public SimpleHDFSConfigStoreFactory(Config factoryConfig) {
```

Comment:
When is this constructor used by the `ConfigClient`? The `ConfigStoreFactory`s are loaded by `ServiceLoader` in `gobblin.config.client.ConfigStoreFactoryRegister`. And it looks like it uses the default no args constructor.